packages/firestore/src/local/indexeddb_remote_document_cache.ts

```diff
@@ -583,10 +582,9 @@ class IndexedDbRemoteDocumentChangeBuffer extends RemoteDocumentChangeBuffer {
 function documentGlobalStore(
   txn: PersistenceTransaction
 ): SimpleDbStore<DbRemoteDocumentGlobalKey, DbRemoteDocumentGlobal> {
-  return IndexedDbPersistence.getStore<
-    DbRemoteDocumentGlobalKey,
-    DbRemoteDocumentGlobal
-  >(txn, DbRemoteDocumentGlobal.store);
+  return txn.getStore<DbRemoteDocumentGlobalKey, DbRemoteDocumentGlobal>(
```

Comment:
This seems like a regression in terms of tree-shakeability. Should this be a free function?